{"name": "test_login_user_not_found", "status": "passed", "attachments": [{"name": "log", "source": "40807305-6624-4ca4-99e5-066e88a6119d-attachment.txt", "type": "text/plain"}], "start": 1754626781157, "stop": 1754626782634, "uuid": "6ac1994e-c60a-40bc-a271-0cd22847c62a", "historyId": "7796ca1f41588f4037c87602ef8c561f", "testCaseId": "7796ca1f41588f4037c87602ef8c561f", "fullName": "Testcases.test_login.TestLogin#test_login_user_not_found", "labels": [{"name": "parentSuite", "value": "Testcases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "VH1347-QC"}, {"name": "thread", "value": "1728-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Testcases.test_login"}]}